t/load_csv_or_excel_file.t

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/perl -w
+
+use strict;
+use warnings;
+use utf8;
+
+use Test::More;
+use Log::Any::Adapter 'TAP';
+
+use File::Basename "dirname";
+
+use ProductOpener::Producers qw(load_csv_or_excel_file);
+
+my $inputs_dir = dirname(__FILE__) . "/inputs/";
```

Comment:
Good idea to add tests, they are indeed badly missing.

Maybe add the name of the test in the directory, like we do for t/expected_test_results, as we might have other types of inputs later.